src/components/common/index.css

```diff
@@ -14,6 +14,10 @@
     0 30px 22px rgba(40, 17, 126, 0.086), 0 100px 80px rgba(92, 78, 129, 0.12);
 }
 
+nav {
```

Comment:
don't use plain html tags as css rules generally speaking. they will clash potentially with other people's nav in case theres another on the page, it's not future proof. give this a class or something